.github/workflows/build.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           -DBUILD_DEV_PLUGINS:BOOL=${{ matrix.plugins == 'all' }} \
           -DBUILD_SIZECHECK:BOOL=${{ matrix.plugins == 'all' }} \
           -DBUILD_SKELETON:BOOL=${{ matrix.plugins == 'all' }} \
-          -DBUILD_STONESENSE:BOOL=${{ matrix.plugins == 'all' }} \
+          -DBUILD_STONESENSE:BOOL=1 \
```

Comment:
What I meant was add `-DBUILD_STONESENSE:BOOL=1 \` to the _Windows_ section, which starts around line 128 right now.

We want to keep this line how it was before because we decided we don't need to build Stonesense for every permutation of our Linux builds - just checking that it compiles once on Linux is fine.

Reply:
That makes sense. thats why i was asking why that line exists.